addons/mail/static/tests/tours/mail_message_load_order_tour.js

```diff
@@ -30,7 +30,8 @@ registry.category("web_tour.tours").add("mail_message_load_order_tour", {
             // the thread service. Thus, at first load the message range
             // will be (31 - 60). This trigger ensures the next messages
             // are fetched after jumping to the message.
-            trigger: ".o-mail-Thread .o-mail-Message:first:not(:contains(31))",
+            trigger:
+                ".o-mail-Thread .o-mail-Message:first .o-mail-Message-textContent:not(:contains(31))",
```

Comment:
I would consider making the message content more explicit in the test setup, rather than complexifying the selector.

And more importantly, I would much prefer if we had exact text match rather than loose contains.